.github/workflows/ci.yml

```diff
@@ -13,6 +13,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v5
+
+      - name: Build libhimmelblau
+        # The code sanity check fails if himmelblau.h does not exist, so we generate it first.
+        run: go generate ./internal/providers/msentraid/...
```

Comment:
Maybe move this commit earlier, when the library is added (and before we use it in code)

Reply:
done